UPGRADING

```diff
@@ -280,6 +280,9 @@ PHP 7.1 UPGRADE NOTES
   . Timezone initialization failure from serialized data will now throw an
     instance of Error from __wakeup() or __set_state() instead of resulting in
     a fatal error.
+  . DateTime instances now support microseconds. Please note this can have impact
+    when comparing; eg. this would previously return TRUE, but now returns FALSE:
+     (new DateTime('first day of next month') == new DateTime('first day of next month'));
```

Comment:
This is not a new change. This could result in false before if the first and second instantiation spanned a change in current second.